datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxBackupLocationValidationRequest.java

```diff
@@ -0,0 +1,30 @@
+package com.sequenceiq.sdx.api.model;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SdxBackupLocationValidationRequest {
+
+    @NotNull
+    @ApiModelProperty(ModelDescriptions.DATA_LAKE_NAME)
+    private String clusterName;
```

Comment:
Perhaps add `@ValidStackNameFormat @ValidStackNameLength` as well that used to be included in the original `SdxEndpoint.validateBackupStorage()` signature. (Actually, when these were added, `@NotNull` would likely become superfluous and could be dropped.)
CC @bbihari

Reply:
I will add. Seeing the usage of @ValidStackNameFormat @ValidStackNameLength, they are followed by @NotNull as well.

Reply:
Most of the places where @ValidStackNameFormat @ValidStackNameLength is applied also has @NotNull.

Reply:
I just test the changed with out @NotNull. I see that this annotation is not needed.
@ValidStackNameLength has below check which should be good.
The length of the name has to be in range of 5 to 40